specification/Metadata/README.md

```diff
@@ -372,64 +316,40 @@ This example shows how to encode a variable-length array of integers.
 
 <img src="figures/array-of-ints.png"  alt="Variable-length array" width="640px">
 
-```jsonc
-{
-  "schema": {
-    "classes": {
-      "arrayExample": {
-        "properties": {
-          "arrayProperty": {
-            "type": "ARRAY",
-            "componentType": "UINT8"
-          }
-        }
-      }
-    }
-  },
-  "entityTable": {
-    "count": 5,
-    "class": "arrayExample",
-    "properties": {
-      "arrayProperty": {
-        "buffer": {...},
-        "arrayOffsetBuffer": {...}
-      }
-    }
-  }
-}
-```
+**schema → classes → arrayExample**
+
+| property        | type    | componentType |
+|-----------------|---------|---------------|
+| "arrayProperty" | "ARRAY" | "UINT8"       |
+
+**table**
+
+- count: `5`
+- class: `"arrayExample"`
+- properties
+  - `"arrayProperty"`
+    - buffer: `[1, 3, 5, ...]`
+    - arrayOffsetBuffer: `[0, 3, 7, ...]`
 
 This example shows how to encode a variable-length array of strings.
 
 ![Variable-length array of string](figures/array-of-strings.png)
 
-```jsonc
-{
-  "schema": {
-    "classes": {
-      "weather": {
-        "properties": {
-          "forecastHistory": {
-            "type": "ARRAY",
-            "componentType": "STRING"
-          }
-        }
-      }
-    }
-  },
-  "entityTable":  {
-    "count": 2,
-    "class": "weather",
-    "properties": {
-      "forecastHistory": {
-        "buffer": {...},
-        "arrayOffsetBuffer": {...},
-        "stringOffsetBuffer": {...}
-      }
-    }
-  }
-}
-```
+**schema → classes → weather**
+
+| property          | type    | componentType |
+|-------------------|---------|---------------|
+| "forecastHistory" | "ARRAY" | "STRING"      |
+
+**table**
+
+- count: `2`
+- class `"weather"`
+- properties
+  - `"forecastHistory"`
+    - buffer: `[R, a, i, n, ...]`
```

Comment:
Maybe be more explicit that the buffer stores the individual characters? not sure the best way to indicate this, here's one idea:

```suggestion
    - buffer: `['R', 'a', 'i', 'n', ...]`
```